src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx

```diff
@@ -42,33 +43,28 @@ import {
 
 export default function FundingCycleDetails({
   fundingCycle,
+  distributionLimit,
+  distributionLimitCurrency,
 }: {
   fundingCycle: V2FundingCycle | undefined
+  distributionLimit: BigNumber | undefined
+  distributionLimitCurrency: BigNumber | undefined
 }) {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
 
-  const { tokenSymbol, projectId, primaryTerminal } =
-    useContext(V2ProjectContext)
-
-  const { data: distributionLimitData } = useProjectDistributionLimit({
```

Comment:
This caused a bug where the project preview didnt update when changing limit.

Kinda unfortunate consequence of our architecture - it's hard to know when to fetch something and when not to.

Anyway, passing in dist limit as props fixes it

Reply:
Nice one ok